bundle/edu.gemini.dataman.app/src/main/scala/edu/gemini/dataman/core/package.scala

```diff
@@ -33,7 +36,10 @@ package object core {
     def fail(f: => DmanFailure): DmanAction[Nothing] = EitherT(Task.delay(f.left))
 
     def mergeFailure[A](result: Throwable \/ TryDman[A]): TryDman[A] =
-      result.fold(ex => DmanFailure.DmanException(ex).left, identity)
+      result.fold({
+        case ie: InterruptedException => throw ie
```

Comment:
By handling `InterruptedException` and turning it into a left, as I was doing here before, we handle the interrupt and the thread continues to run.  I'd need to check everywhere whether the `DmanFailure` is due to an `InterruptedException`.  Admittedly though, I'm not completely sure this is the right thing to do.